### PR TITLE
Polkadot v0.9.38

### PR DESCRIPTION
**Pull request by cdamian**

### CODE REVIEW

**Review by mustermeiszer — APPROVED** [review]

File: core/src/tests/parachain.rs [hunk 1]
```diff
@@ -124,9 +124,6 @@ fn cidp_and_dp_relay(
 			let parent_header = client.header(parent).unwrap().unwrap();
 
 			async move {
-				let uncles =
-					sc_consensus_uncles::create_uncles_inherent_data_provider(&*client, parent)?;
-
```
Comment (mustermeiszer) [inline core/src/tests/parachain.rs:129]: No more uncles? Can you check if the polkadot repo -- the polkadot runtime -- also does not expect uncles anymore.

File: integration_test/src/main.rs [hunk 2]
```diff
@@ -35,7 +35,6 @@ type RCidp = Box<
 		InherentDataProviders = (
 			FudgeInherentTimestamp,
 			sp_consensus_babe::inherents::InherentDataProvider,
-			sp_authorship::InherentDataProvider<RHeader>,
```
Comment (mustermeiszer) [inline integration_test/src/main.rs:38]: I am not sure. 

What did `sp_authorship` provide?

File: core/src/tests/parachain.rs (hunk 1 above)
Comment (cdamian) [reply core/src/tests/parachain.rs:129]: Yup, it definitely seems like it - https://github.com/paritytech/substrate/blob/polkadot-v0.9.38/frame/authorship/src/lib.rs#L44

Comment (cdamian) [reply core/src/tests/parachain.rs:129]: Ah, lemme also check the runtime.

Comment (cdamian) [reply core/src/tests/parachain.rs:129]: No uncles mentioned here as well - https://github.com/paritytech/polkadot/blob/release-v0.9.38/runtime/polkadot/src/lib.rs

Comment (cdamian) [reply core/src/tests/parachain.rs:129]: Gotta dig deeper into the issue described here - https://github.com/paritytech/polkadot/blob/release-v0.9.37/runtime/polkadot/src/lib.rs#L340

Comment (mustermeiszer) [reply core/src/tests/parachain.rs:129]: Interesting... I thought they were relevant for consensus, hence my question.

**Comment by cdamian** [comment]
Draft until the `uncles` situation is clarified. The tests are passing so I am assuming we are not covering this in these tests @mustermeiszer?

**Review by mustermeiszer — APPROVED** [review]

**Pull request merged**